src/app/pages/tables/components/hotTables/handsontable-section.ts

```diff
@@ -0,0 +1,41 @@
+import { Component } from '@angular/core';
+ 
+@Component({
+   selector: 'handsontable-section',
+   template: `
+   <section id="handsontable">
+     <div class="row">
+          <div class="col-md-12">
+                     <basic-demo></basic-demo>
+                     <br>
+                     <br>
+                     <advanced-demo></advanced-demo>
+                     <br>
+                     <br>
+                     <finance-demo></finance-demo>  
+                     <br>
+                     <science-demo></science-demo>
+                     <br>
+                     <br>   
+                     <sheet-demo></sheet-demo>
+                     <br>
+                     <br>
+                     <sport-demo></sport-demo>
+         </div>
+     </div>
+   </section>
+   `
+})
+export class HandsontableSection {
```

Comment:
A component name should have `Component` prefix. For example `HandsontableSectionComponent`.